### DISCUSSION TITLE
Clustering

### DISCUSSION THREAD

**Discussion by WildArne** [deeztek/Hermes-Secure-Email-Gateway#12] [Support] [answered]
Can it do clustering? :-)

**Comment by deeztek** [comment, marked as answer]
HA is on the roadmap to be implemented on the next version of Hermes SEG. The problem with HA is not as simple as simply replicating MySQL db and/or filesystem. It's going to require the development of an API because there are a lot of background things that happen when you perform seemingly simple tasks such as adding users.

**Reply by WildArne** [reply]
Awesome, i will be waiting for it :-)
